db/migrate/20230228143905_create_users.rb

```diff
@@ -2,12 +2,12 @@ class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
       t.string :name
-      t.string :email
-      t.string :password_digest
+      t.string :Bio
+      t.string :post_counter
       t.string :photo_link
 
       t.timestamps
     end
-    add_index :users, :email, unique: true
+    add_index :users, unique: true
```

Comment:
* AS Stated by the previous CR your table and column names should match the ERD diagram, kindly change the foreign key column from `user_id` to `author_id`